web-api/src/persistence/postgres/utils/getColumnsForTable.ts

```diff
@@ -0,0 +1,5 @@
+import { Database, DatabaseSchema } from '@web-api/database-types';
+
+export const getColumnsForTable = (table: keyof Database): any[] => {
```

Comment:
I am experimenting with ways to improve the typing. This function is currently only used in `pgInsertInto` and typed this way because kysely expects a certain (fairly complex) type for `updateSet`. The idea here is that `pgInsertInto` can then automatically infer columns rather than the user always specifying which columns to update. (The latter is better practice, but the former more faithfully translates our current "upsert everything" approach.)

These columns are essentially just strings, but Kysely wraps them in a custom Column type. In a quick experiment, I managed to make the typing here "better" (e.g., not "any"), but it leads to more convoluted, kysely-specific typing that still had a clunky bit I couldn't fully resolve. Rather than go down this rabbit hole further, I aborted.